hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/SCMSecurityProtocolServer.java

```diff
@@ -93,15 +100,22 @@ public class SCMSecurityProtocolServer implements SCMSecurityProtocol {
   private final ProtocolMessageMetrics metrics;
   private final StorageContainerManager storageContainerManager;
 
+  // SecretKey may not be enabled if when neither block token nor container
```

Comment:
typo in comment, you can remove "if"
```suggestion
  // SecretKey may not be enabled when neither block token nor container
```